pkg/alert/crdinstaller.go

```diff
@@ -170,7 +170,7 @@ func (c *CRDInstaller) AddInformerEventHandler() {
 
 // CreateHandler will create a CRD handler
 func (c *CRDInstaller) CreateHandler() {
-	routeClient := util.GetRouteClient(c.kubeConfig)
+	routeClient, _ := util.GetRouteClient(c.kubeConfig)
```

Comment:
Since the error is not handled in any of the places, remove the error from the return

Reply:
GetRouteClient logs errors instead of returning them